src/sentry/workflow_engine/models/alertrule_workflow.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import CheckConstraint, Q, UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+
+
+@region_silo_model
+class AlertRuleWorkflow(DefaultFieldsModel):
+    """
+    A lookup model for rules and workflows.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    alert_rule = FlexibleForeignKey("sentry.AlertRule", null=True)
+    rule = FlexibleForeignKey("sentry.Rule", null=True)
+    workflow = FlexibleForeignKey("workflow_engine.Workflow")
```

Comment:
can we put the detector here too?

Reply:
Can't you look that up through the `DetectorWorkflow` model?

Reply:
hmm i guess so. thinking about it more, even if a workflow is attached to multiple detectors, the lookup table will have the same function as DetectorWorkflow and can't help you distinguish which one.

perf detectors and error detectors can have same workflow even before we GA ACI because issue alerts applies to all issue types rn